pkg/exchange/max/maxapi/trade.go

```diff
@@ -111,129 +116,36 @@ func (options *QueryTradeOptions) Params() url.Values {
 	return params
 }
 
-func (s *TradeService) NewPrivateTradeRequest() *PrivateTradeRequest {
-	return &PrivateTradeRequest{client: s.client}
+func (s *TradeService) NewGetPrivateTradeRequest() *GetPrivateTradesRequest {
+	return &GetPrivateTradesRequest{client: s.client}
 }
 
 type PrivateRequestParams struct {
 	Nonce int64  `json:"nonce"`
 	Path  string `json:"path"`
 }
 
-type PrivateTradeRequest struct {
-	client *RestClient
+//go:generate GetRequest -url "v2/trades/my" -type GetPrivateTradesRequest -responseType []Trade
+type GetPrivateTradesRequest struct {
```

Comment:
no page number here?

Reply:
yeah, we don't use pagination for this request